frontend/src/components/DateRangePicker.js

```diff
@@ -15,11 +15,25 @@ const phrases = {
 };
 
 function DateRangePicker({
-  query, onUpdateFilter, id,
+  query, onUpdateFilter, id, classNames, gainFocus,
 }) {
   const [focusedInput, updateFocused] = useState(null);
   const [opened, updateOpened] = useState(false);
 
+  const startDateId = `${id}-start`;
+  const endDateId = `${id}-end`;
+
+  useEffect(() => {
+    /** *
+     * This is for when the date range selector is conditionally created
+     * It will auto focus the start date input
+     */
+    if (gainFocus) {
```

Comment:
This is specifically to transfer focus from the "Date range picker" to the newly rendered react-date component.